src/test/java/io/supertokens/test/passwordless/api/PasswordlessConsumeCodeAPITest2_11.java

```diff
@@ -249,9 +248,7 @@ public void testBadInput() throws Exception {
 
             assertNotNull(error);
             assertEquals(400, error.statusCode);
-            assertEquals(
-                    "Http error. Status Code: 400. Message: Please provide exactly one of linkCode or deviceId+userInputCode",
-                    error.getMessage());
+            assertEquals("Http error. Status Code: 400. Message: Input encoding error in LinkCode", error.getMessage());
```

Comment:
Error message should not have capital "L" in "LinkCode". Since the input is "linkCode". If this requires a change in the API, please do so in this PR.

Reply:
noted.